### PR TITLE
Add new CNCF case study to Knative website

### LINKED ISSUE #5986: Add new CNCF case study to Knative website
https://www.cncf.io/case-studies/ibmwatsonxassistant/

Tasks:

- Place it on https://knative.dev/docs/ homepage:
Screenshot 2024-05-22 at 11 15 44

- Place it on https://knative.dev/docs/about/case-studies/ page
- Add a separate page, similar to https://knative.dev/docs/about/case-studies/sva/

### PR DESCRIPTION
**Pull request by rohitsrma**
Fixes #5986 

### Homepage

image

### Case studies list

image

### IBM case study page

image

### CODE REVIEW

File: overrides/home.html [hunk 1]
```diff
@@ -236,6 +236,20 @@ <h1>
...
+            </div>
+            <hr>
+            <p class="normal-text">
+              "The introduction of the ML training infrastructure using Knative Eventing 
+              has enabled us to establish a well-defined operational boundary for the service teams."<br><br>
+
+              —IBM
+            </p>
+          </a>
+        </div>
```
Comment (aliok) [inline overrides/home.html:252]: Let's only show 3 case studies in home page.

Can you check the dates of the case studies and update the homepage to show the latest 3?

File: docs/about/case-studies/ibm.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<table style="border: 0;">
+<tr style="background-color: var(--md-default-bg-color);">
+<td style="border: 0;">
+<div style="min-width: 20mm;">
+      <img src="../../../images/case-studies/ibm.png" alt="IBM logo" draggable="false" />
+</div>
+<div>
+<em style="color:#0071c7;font-size: 1em;">“We were looking for a solution that would be simple enough to maintain while providing 100% ownership to the service team in all aspects. We prototyped a system in early 2022 using Knative Eventing (backed by Knative Kafka Broker) for our watsonx Assistant use-case. Our initial results exceeded our existing benchmarks at various levels. After investing enough time to make it production ready, we rolled it out across all production IBM cloud clusters in six geographical regions. After successfully handling hundreds of thousands of ML model trainings in this initial rollout period and monitoring the system’s resiliency and impact, we also upgraded our watsonx Assistant on CloudPak for Data and watsonx Orchestrate on AWS Marketplace offerings to utilize Knative Eventing for machine learning training via RedHat Openshift Serverless offering.”
```
Comment (aliok) [inline docs/about/case-studies/ibm.md:13]:
```suggestion
<em style="color:#0071c7;font-size: 1em;">“We were looking for a solution that would be simple enough to maintain while providing 100% ownership to the service team in all aspects. We prototyped a system in early 2022 using Knative Eventing (backed by Knative Kafka Broker) for our watsonx Assistant use-case. Our initial results exceeded our existing benchmarks at various levels. After investing enough time to make it production ready, we rolled it out across all production IBM cloud clusters in six geographical regions.”
```

Let's shorten this part a little.

File: overrides/home.html [hunk 3]
```diff
@@ -236,6 +236,20 @@ <h1>
...
+          <a href="{{ 'about/case-studies/ibm/' | url }}">
+            <div class="logo">
+              <img class="ibm" src="images/case-studies/ibm.png" alt="IBM logo" draggable="false" width="75" />
+            </div>
+            <hr>
+            <p class="normal-text">
+              "The introduction of the ML training infrastructure using Knative Eventing 
+              has enabled us to establish a well-defined operational boundary for the service teams."<br><br>
+
+              —IBM
```
Comment (aliok) [inline overrides/home.html:249]:
```suggestion
```
Let's remove the --IBM text here

**Review by aliok — COMMENTED** [review]
Thanks @rohitsrma .

There are some changes I've requested

File: overrides/home.html (hunk 1 above)
Comment (rohitsrma) [reply overrides/home.html:252]: @aliok The existing three case studies are not available on the [CNCF website](https://www.cncf.io/case-studies?_sft_lf-project=knative). Could you please tell me where these case studies were published?

I have verified that the [PR](https://github.com/knative/docs/pull/4406) to add these was merged in 2021, so all four case studies on the CNCF website are newer than these three.

File: docs/about/case-studies/ibm.md (hunk 2 above)
Comment (rohitsrma) [reply docs/about/case-studies/ibm.md]: Done!

**Comment by rohitsrma** [comment]
Hello @aliok, I've made the changes. Just have a little doubt [here](https://github.com/knative/docs/pull/5987#discussion_r1611069406).

File: overrides/home.html (hunk 1 above)
Comment (aliok) [reply overrides/home.html]: You're right.

Then can you identify the latest ones from the PR dates? Let's show the latest 3 only.

**Comment by rohitsrma** [comment]
Hii @aliok, I've made the changes to display the latest three case studies on the homepage. These case studies are also listed on the [CNFC website](https://www.cncf.io/case-studies/?_sft_lf-project=knative).

image

Please let me know if any further changes are needed.

**Comment by aliok** [comment]
Thanks a lot @rohitsrma 

Merging this now!

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**